modules/ROOT/pages/constraints/managing-constraints.adoc

```diff
@@ -0,0 +1,1598 @@
+----
+
+The constraint creation fails on the first offending relationship that is found.
+This does not guarantee that there are no other offending relationships in the graph.
+Therefore, all the data should be checked and cleaned up before re-attempting the constraint creation.
+
+
+.Find all offending relationships missing the property for the constraint above
+[source, cypher]
+----
+MATCH ()-[wrote:WROTE]-()
+WHERE wrote.language IS NULL
+RETURN wrote
+----
+
+======
+
+[[removing-an-existing-constrained-property-will-fail]]
+==== Removing existence constrained properties will fail
+
+.Remove a node property existence-constrained property
+======
+
+.Remove the `name` property from an existing `Author` node, given a property existence constraint on `:Author(name)`
+[source, cypher, role=test-fail]
+----
+MATCH (author:Author {name: 'Virginia Woolf'})
+REMOVE author.name
+----
+
+In this case, the property is not removed because it is in conflict with an existing constraint.
+
+.Error message
+[source, error]
+----
+Node(0) with label `Author` must have the property `name`
+----
+
+======
+
+
+.Remove a relationship property existence-constrained property
+======
+
+////
+[source, cypher, role=test-setup]
+----
+CREATE CONSTRAINT wrote_locations FOR ()-[wrote:WROTE]-() REQUIRE wrote.location IS NOT NULL
+----
+////
+
+.Remove the `location` property from an existing `WROTE` relationship, given a property existence constraint `:WROTE(location)`
+[source, cypher, role=test-fail]
+----
+MATCH (author:Author)-[wrote:WROTE]->(book:Book)
+REMOVE wrote.location
+----
+
+In this case, the property is not removed because it is in conflict with an existing constraint.
+
+.Error message
+[source, error]
+----
+Relationship(0) with type `WROTE` must have the property `location`
+----
+
+======
+
+[[create-key-and-uniqueness-constraint-on-same-schema-as-existing-index]]
+==== Creating key and uniqueness constraints on the same schema as an existing index will fail
+
+
+.Creating a node property uniqueness constraint on the same name as an existing index
+======
+
+////
+[source, cypher, role=test-setup]
+----
+CREATE INDEX preExisting_book_word_count FOR (book:Book) ON (book.wordCount)
+----
+////
+
+.Create a constraint requiring all `Book` nodes to have unique `wordCount` properties, when an index already exists on that label and property combination
+[source, cypher, role=test-fail]
+----
+CREATE CONSTRAINT book_word_count FOR (book:Book) REQUIRE book.wordCount IS UNIQUE
+----
+
+In this case, the constraint cannot be created because there already exists an index covering that schema.
+
+.Error message
+[source, error]
+----
+There already exists an index (:Book {wordCount}).
+A constraint cannot be created until the index has been dropped.
+----
+
+======
+
+
+.Creating a relationship property key constraint on the same name as an existing index
```

Comment:
```suggestion
.Creating a relationship key constraint on the same name as an existing index
```

Reply:
this example also actually shows the same name case which is nice to have but I think the section title might need updating to not just say `same schema` since we're showing more than that